package.json

```diff
@@ -58,13 +58,13 @@
     "eslint-plugin-react": "^7.14.3",
     "eslint-plugin-standard": "^4.0.1",
     "extend": "~3.0.0",
-    "extract-text-webpack-plugin": "^4.0.0-beta.0",
     "html-webpack-plugin": "^3.2.0",
     "htmlparser": "~1.7.7",
     "imports-loader": "~0.6.5",
     "jed": "~1.1.0",
     "less": "~3.0.1",
     "less-loader": "~4.0.6",
+    "mini-css-extract-plugin": "0.8.0",
```

Comment:
I think we should use ^ here. Not all our devDependencies use that (particularly not the older ones), but over time this is the most sensible default.